nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/users/feature/users.component.spec.ts

```diff
@@ -34,7 +35,13 @@ describe('Users', () => {
     beforeEach(() => {
         TestBed.configureTestingModule({
             declarations: [Users, UserListing],
-            imports: [RouterModule, RouterTestingModule, MockComponent(BannerText), MockComponent(Navigation)],
+            imports: [
+                RouterModule,
+                RouterTestingModule,
+                MockComponent(BannerText),
+                MockComponent(Navigation),
+                MockComponent(NgxSkeletonLoaderComponent)
```

Comment:
I don't think this component uses the skeleton loader so mocking here shouldn't be needed. I realize this PR been open for some time so it's possible the components changed since opening.